reconcile/utils/terrascript_aws_client.py

```diff
@@ -5376,6 +5376,25 @@ def populate_tf_resource_alb(self, spec, ocm_map=None):
                 f"{identifier}-{target_name}", **lbt_random_id_values
             )
             tf_resources.append(lbt_random_id)
+            health_check_interval = 10
+            health_check_timeout = 5
+            health_check_unhealthy_threshold = 3
+            health_check_healthy_threshold = 3
+            if health_check := t.get("health_check"):
```

Comment:
will we ever get `t.get("health_check") == None` ?

Reply:
Its an optional value in the schema. My understanding is, that this will return `None`. The intention of this check is to only act if `health_check` is configured. But probably Im misinterpreting the question?

Reply:
Since we query
```
 health_check {
    unhealthy_threshold
    timeout
    interval
    healthy_threshold
}
```
I think we will never have `health_check` set to `None` but we can have its fields set to `None` (as tested below).

This is really a nit, but I'd like to make sure I am not wrong :wink:

Reply:
Ahh I understand.

Reconfirmed in the debugger now - `health_check` is Ǹone` if its not specified in app-interface.

Reply:
Ack